### ISSUE TITLE
Minor nit: Graphics overflow in upgrader substep progress bar

### ISSUE THREAD

**Issue by sbulen** [SimpleMachines/SMF#4779]
Someone had recently fixed the progress bars in the upgrader.  
I just noticed that the substep progress bar needs a similar fix...
Noticeable mainly when doing the long-running attachments step.
See photo...  
![image](https://user-images.githubusercontent.com/23568484/40879667-3f936434-6658-11e8-942b-843df7d210ba.png)

**Comment by Gwenwyfar** [comment]
While testing this I noticed firefox was blocking the images and css files from upgrader (but not from installer). Another bug perhaps? I have https enabled.

**Comment by illori** [comment]
did you inspect the page to see what URL was being called for the css?

**Comment by Gwenwyfar** [comment]
Good catch. The URLs are being generated as http. I'll open an issue.

**Comment by illori** [comment]
Can you open a separate issue for that?

**Comment by Gwenwyfar** [comment]
I already have a fix for this, I'm submitting it after #4791 has been merged.
